about.html

```diff
@@ -0,0 +1,103 @@
+						<li><a href="#">Program</a></li>
+						<li><a href="#">Join</a></li>
+						<li><a href="#">Sponsor</a></li>
+						<li><a href="#">News</a></li>
+						<li><a href="#" id="apply" >APPLY Now</a></li>
+					</ul>
+				</div>
+				<img src="/img/home/hamburgermenu.svg" alt="burgermenu" class="burger">
+			</nav>
+		</header>
+		<main class="card about">
+			<header class="card-header">
+				<h1 class="card-heading">
+					ARE YOU READY TO BE COME A SOFTWARE DEVELOPER AND WORK REMOTELY FOR GLOBAL COMPANIES 2022?
+				</h1>
+			</header>
+			<div class="card-body">
+				<p class="card-title">
+          Microverse is an online school emphasizing community, accountability, and support, fostering a great place to learn to code remotely.
+					Microverse students are from countries around the world.
+          At Microverse, you learn with a diverse, driven community and connect to life-changing job opportunities.
+          Gain the skills you need to get hired. Learn online, connect to life-changing opportunities. Pay $0 until you land a job!
+				</p>
+			</div>
+			<div class="about-card-footer">
+				<p class="card-contactUs">
+          Talent is everywhere, but opportunity is not.
+         connect one million people to remote jobs by 2030.
+					Please contact us through at address
+					<a href="#">mara234216@gmail.com</a> or apply
+          <a href="https://www.microverse.org
+          ">https://www.microverse.org
+        </a>
+				</p>
+			</div>
+		</main>
+		<section class="about-program flex-column">
+			<h4 class="about-program-heading">
+				MICROVERSE <span>2022</span> LOGO APPLY NOW
+			</h4>
+			<p class="about-program-text">
+				This is  Microverse Logo
+			</p>
+			<a href="#" class="logo-about flex-row"><img src="/img/Microverse.jpg" alt="Microverseimg">
+			</a>
+		</section>
+    <section id="microverse-annual">
+      <h4 class="last-summit-heading">
+				See the past microverseInc's <span>annu</span>al team retreat in Florence.
+			</h4>
+      <div class="conference-container">
+        <div id="annual">
+          <h2>2022</h2>
+          <h3>Microverse annual team retreat in Florence. 50+ people from more than 20 countries and 5 continents were represented. </h3>
+        </div>
+        <div id="team">
+          <h2>2021</h2>
+          <h3>Microverse is transforming global developer education with a unique ISA model and peer-to-peer training modality.</h3>
+        </div>
+      </div>
+    </section>
+		<section class="partners flex-column">
+			<h4 class="partners-heading">P<span>artn</span>ers</h4>
+			<ul class="partners-body flex-row">
+				<li><img src="/img/partners/PIA.png" alt="PIA's Icon"></li>
+				<li><img src="/img/partners/Google.png" alt="Google's Icon"></li>
+				<li><img src="/img/partners/Mozilla.png" alt="Mozilla's Icon"></li>
+			</ul>
+		</section>
+		<footer class="footer flex-row">
+			<a href=""><img src="/img/Microverse.jpg" alt=""></a>
+			<p> Join Microverse 2022.</p>
+		</footer>
```

Comment:
- [ ] Your footer here should look similar to that of the original design.
### You
![image](https://user-images.githubusercontent.com/45460192/192003954-017ab769-a96e-4b2c-97af-8d63fc34c3a5.png)
r Implementation

### Required 
![image](https://user-images.githubusercontent.com/45460192/192003876-e181de91-a819-4150-a563-ba5fb7e47e86.png)